iina/PrefKeyBindingViewController.swift

```diff
@@ -434,11 +384,11 @@ extension PrefKeyBindingViewController: NSTableViewDelegate, NSTableViewDataSour
   }
 
   func tableViewSelectionDidChange(_ notification: Notification) {
+    guard !isLoading else { return }
```

Comment:
Seems that `isLoading`'s only usage is to filter out didChange notifications when calling `loadConfigFile`? If so, it's better to add a parameter like `loadConfigFile(_, reloadTable:)`

Reply:
`isLoading` is preventing infinite loops, since we call `loadConfigFile` in `tableViewSelectionDidChange`, and we also call `reloadData` in `loadConfigFile`, which will cause another `tableViewSelectionDidChange`.